peas/sensors.py

```diff
@@ -150,21 +154,24 @@ def find_arduino_devices():
     return [p.device for p in comports if 'Arduino' in p.description]
 
 
-def auto_detect_port(port, logger):
+def auto_detect_port(port, logger=None):
```

Comment:
I think I was confused by the name of the method. We are not detecting the port but the board type (since you are passing a port).

Reply:
Fixed.